contracts/Removal.sol

```diff
@@ -65,40 +53,85 @@ contract Removal is
     _setApprovalForAll(owner, operator, approved);
   }
 
+  function createRemovalId(bytes calldata removalData)
+    public
+    pure
+    returns (uint256)
+  {
+    return RemovalUtils.createRemovalId(removalData);
+  }
+
+  function unpackRemovalId(uint256 removalId)
+    public
+    pure
+    returns (UnpackedRemovalIdV0 memory)
+  {
+    return removalId.unpackRemovalId();
+  }
+
+  function version(uint256 removalId) public pure returns (uint256) {
+    return removalId.version();
+  }
+
+  function methodology(uint256 removalId) public pure returns (uint256) {
```

Comment:
seems weird to repeat this stuff with this syntactic sugar vs just inlining it or renaming the functions in the library. Can you justify this?

Reply:
per our discussion, we removed everything except `createRemovalId` and `unpackRemovalId` which gets you all the data back at once as a struct